codec-http2/src/test/java/io/netty/handler/codec/http2/Http2MultiplexTest.java

```diff
@@ -1591,17 +1603,40 @@ public void windowUpdatesAreFlushed() {
         // Trigger a read of the second frame.
         childChannel.read();
         verifyFramesMultiplexedToCorrectChannel(childChannel, inboundHandler, 1);
-        // We expect a flush here because the StreamChannel will flush the smaller increment but the
-        // connection will collect the bytes and decide not to send a wire level frame until more are consumed.
-        assertTrue(flushSniffer.checkFlush());
+        if (autoWriteWindowUpdateFrames) {
+            // We expect a flush here because the StreamChannel will flush the smaller increment but the
+            // connection will collect the bytes and decide not to send a wire level frame until more are consumed.
+            assertTrue(flushSniffer.checkFlush());
+        } else {
+            assertFalse(flushSniffer.checkFlush());
```

Comment:
how about `assertEquals(autoWriteWindowUpdateFrames, flushSniffer.checkFlush())` ?

Reply:
I found it a bit more "clear" to use the current style as we need the if / else anyway